exporter/observiqexporter/README.md

```diff
@@ -1,5 +1,11 @@
 # observIQ Exporter
 
+| Status                   |           |
+| ------------------------ |-----------|
+| Stability                | [beta]    |
```

Comment:
Based on line 11, i suspect this should be `in development`, pinging codeowner for confirmation @binaryfissiongames

Reply:
Yeah, `in development` would be more appropriate.

Reply:
ok, change to `in development`